money.gemspec

```diff
@@ -14,6 +14,8 @@ Gem::Specification.new do |s|
   s.description = "A Ruby Library for dealing with money and currency conversion."
   s.license     = "MIT"
 
+
+  s.add_dependency "bigdecimal", "~> 3.1" if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('6.19.0')
```

Comment:
Hello, I have some concerns about this implementation, in example 6.18.0 should be compatible with Ruby 3.4 and it should require bigdecimal.

This is checking that the current RUBY VERSION (3.4 at most) is >= 6.19.0, which will not be the case for the next few years

Reply:
would you please submit an alt. will hold any release until we are all satisfied

Reply:
Doing that 👍🏼